### ISSUE TITLE
OnWindowLoaded() not called when addin is installed

### ISSUE THREAD

**Issue by joemclain** [RickStrahl/MarkdownMonster#847]
For what it's woth, `OnWindowLoaded()` is not called when an addin is first downloaded, installed and then run.

-joe mclain-

**Comment by RickStrahl** [comment]
Addins are not fully loaded through the whole pipeline when installed. Only these events fire:

* OnApplicationStarted()
* OnApplicationInitialized() (OnModelLoaded())

But other events do not fire. For full initialization, a restart is required.

I'm adding `OnWindowLoaded()`. Here's what the full 'manual' load sequence fires:

```cs
if (installedAddin != null)
{
    installedAddin.OnApplicationStart();

    try
    {
        if (mmApp.Model.Window != null)
        {
            // TODO: Remove after OnApplicationInitialized has been around a bit v1.3.20  or so
            installedAddin.OnModelLoaded(mmApp.Model);

            installedAddin.OnApplicationInitialized(mmApp.Model);

            if (mmApp.Model.Window != null) 
                installedAddin.OnWindowLoaded();
        }
    } catch {
        
    }

    // trigger parser addins to refresh MarkdownParsers if they are provided
    if (installedAddin.GetMarkdownParser(false, false) != null)
        mmApp.Model.OnPropertyChanged(nameof(AppModel.MarkdownParserNames));

    InitializeAddinsUi(mmApp.Model.Window,
        new List<MarkdownMonsterAddin>()
        {
            installedAddin
        });                            
}
}
```

**Comment by RickStrahl** [comment]
Fixed in 2.3.17+
